src/unix/stream.c

```diff
@@ -301,7 +301,7 @@ int uv__stream_try_select(uv_stream_t* stream, int* fd) {
   if (fds[1] > max_fd)
     max_fd = fds[1];
 
-  sread_sz = (max_fd + NBBY) / NBBY;
+  sread_sz = ROUND_UP((max_fd + 1) / NBBY, sizeof(uint32_t));
```

Comment:
Isn't this wrong when max_fd is, let's say, 32?  `ROUND_UP((32 + 1) / 8, sizeof(uint32_t))` is 4 when it should be 8.